dev/Common/IsWindowsVersion.h

```diff
@@ -15,7 +15,9 @@ inline bool IsExportPresent(
     wil::unique_hmodule dll{ LoadLibraryExW(filename, nullptr, 0) };
     if (!dll)
     {
-        THROW_LAST_ERROR();
+        DWORD getLoadLibraryError = GetLastError();
```

Comment:
NIT: `rcLoadLibrary`

GetLastError = Win32 error code. Common short variable name for that is `rc` (short for `return code`), much like `hr` is the common name for an HRESULT.

If you want multiple `rc` variables then `rcFoo` and `rcBar` is the common pattern, or in this case `rcLoadLibrary` and `rcGetProcAddress`